src/test/regress/sql/gp_explain.sql

```diff
@@ -329,3 +329,15 @@ CREATE TABLE t1 (a int);
 EXPLAIN ANALYZE INSERT INTO t1 SELECT 1/gp_segment_id
 FROM gp_dist_random('gp_id');
 DROP TABLE t1;
+
+
+
+create table foo_alias (a int, b int);
```

Comment:
You should first use `drop table if exist` in `begin_ignore` and `end_ignore` block